draft-ietf-gnap-core-protocol.md

```diff
@@ -5458,6 +5472,184 @@ This specification requests registration of the following scheme in the
+* Intended usage: COMMON
+* Restrictions on usage: none
+* Author: IETF GNAP Working Group, txauth@ietf.org
+* Change Controller: IETF
+* Provisional registration?  No
+
+To indicate that the content is a GNAP token rotation message to be bound with a detached JWS mechanism:
+
+* Type name: application
+* Subtype name: gnap-binding-rotation+jwsd
+* Required parameters: n/a
+* Optional parameters: n/a
+* Encoding considerations: binary
+* Security considerations: See {{security}} of {{&SELF}}
+* Interoperability considerations: n/a
+* Published specification: {{&SELF}}
+* Applications that use this media type: GNAP
+* Fragment identifier considerations: n/a
+* Additional information:
+  * Magic number(s): n/a
+  * File extension(s): n/a
+  * Macintosh file type code(s): n/a
+* Person & email address to contact for further information: IETF GNAP Working Group, txauth@ietf.org
+* Intended usage: COMMON
+* Restrictions on usage: none
+* Author: IETF GNAP Working Group, txauth@ietf.org
+* Change Controller: IETF
+* Provisional registration?  No
+
+To indicate that the content is a GNAP token rotation message to be bound with an attached JWS mechanism:
+
+* Type name: application
+* Subtype name: gnap-binding-rotation+jws
+* Required parameters: n/a
+* Optional parameters: n/a
+* Encoding considerations: binary
+* Security considerations: See {{security}} of {{&SELF}}
+* Interoperability considerations: n/a
+* Published specification: {{&SELF}}
+* Applications that use this media type: GNAP
+* Fragment identifier considerations: n/a
+* Additional information:
+  * Magic number(s): n/a
+  * File extension(s): n/a
+  * Macintosh file type code(s): n/a
+* Person & email address to contact for further information: IETF GNAP Working Group, txauth@ietf.org
+* Intended usage: COMMON
+* Restrictions on usage: none
+* Author: IETF GNAP Working Group, txauth@ietf.org
+* Change Controller: IETF
+* Provisional registration?  No
+
+##  Structured Syntax Suffix Registration
+
+This section requests registration of the "+jws" and "+jwsd" structured syntax suffixes in
+the "Structured Syntax Suffix" registry {{IANA.StructuredSuffix}} in
+the manner described in {{RFC6838}}, which can be used to indicate that
+the media type is encoded as a compact form JWS with either attached or detached payload content.
+
+To indicate the content is a compact form JWS as defined by {{RFC7515}} with attached payload:
+
+* Name: JWS
+* +suffix: +jws
+* References: {{&SELF}}
+* Encoding considerations: binary; JWS values are a series of base64url-encoded values (some of which may be the empty string) separated by period ('.') characters.
+* Interoperability considerations: n/a
+* Fragment identifier considerations: n/a
+* Security considerations: See {{security}} of {{&SELF}}
+* Author: IETF GNAP Working Group, txauth@ietf.org
+* Author/Change controller: IETF
+
+To indicate the content is a compact form JWS as defined by {{RFC7515}} with detached payload:
+
+* Name: Detached JWS
```

Comment:
`jwsd` is unnecessary and should be deleted.  Per https://www.rfc-editor.org/rfc/rfc7515.html#appendix-F, you can tell whether the payload is detached or not by whether the payload field is empty or not.